HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -548,6 +549,7 @@
         'keep *_hltIter4Merged_*_*',
         'keep *_hltIterL3GlbMuon_*_*',
         'keep *_hltIterL3MuonMerged_*_*',
+        'keep *_hltIterL3MuonAndMuonFromL1Merged_*_*',
```

Comment:
This is an auto-generated file, thus changes made directly will be overwritten.
You need to file a JIRA ticket requeting addition of this product!

Reply:
I never created a JIRA ticket, and would not know what is to be indicated in several fields... would you please help me ? 
The point is simple: this is the highest-level collection of muon tracker tracks (linked to the reco::Muon object in HLT), which enter in many trigger paths. The validation cannot run for it if it is not saved in the Event. Of course the trigger (or its offline version) will run anyway since it is all happening in the same step. 
Being one of the most important ingredients in the Muon HLT it *should* be validated. 
@Martin-Grunewald @folguera @Fedespring @cericeci @calderona

Reply:
Please remove the changes to this file. 
It is part of https://github.com/cms-sw/cmssw/pull/28681 done through ConfDB

Reply:
Thanks ! doing it